### PR TITLE
Show Google logo to conform to TOS

### PR DESCRIPTION
**Pull request by AndrewLester**
For context, Google sent us a warning to display their logo appropriately. Since its default location is bottom left and would be covered by our UI, I moved it to bottom center. I don't love the location, but I feel like it needs to be away from our UI and other map buttons so it's not confusing.
image

image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -39,10 +39,13 @@ button:active, a.button:active {
     background-color: rgba(189, 29, 37, 0.8);
 }
 /* Google stuff */
-.gmnoscreen, .gmnoprint a, .gm-style-cc,
-div a[rel="noopener"] img {
+.gmnoscreen, .gmnoprint a, .gm-style-cc {
     display: none !important;
 }
+div a[rel="noopener"] img {
+    left: 50vw !important;
```
Comment (ErikBoesen) [inline css/style.css]: Can we use a `left` property corresponding to the actual width of the attribution box?

**Review by ErikBoesen — CHANGES_REQUESTED** [review]
Seems like it would be better to make it aligned with the attribution box.

File: css/style.css (hunk 1 above)
Comment (AndrewLester) [reply css/style.css]: Moved with `10px` space
image

Comment (ErikBoesen) [reply css/style.css]: Can you do it a bit closer, to match the bottom padding?

Comment (AndrewLester) [reply css/style.css]: The image's box is now flush with the attribution box, just a little padding in the image itself:
image

**Review by ErikBoesen — APPROVED** [review]

**Pull request merged**